From c916c8ec597c1c380710a0bc0ab0813c2d6cbda4 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 26 Dec 2017 20:17:42 +0800
Subject: [PATCH 1261/5242] MLK-17293-4 arm: dts: imx7ulp: remove snvs node

commit  c4bf0e8219d3eacaa296a84111fdab2970cac117 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX7ULP B0 chip, SNVS is located in M4 domain,
remove it from dtsi.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index f9b7de0..2bfbe61 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -167,11 +167,6 @@
 			status = "disabled";
 		};
 
-		snvs: snvs@40230000 {
-			compatible = "fsl,sec-v4.0-mon","syscon", "simple-mfd";
-			reg = <0x40230000 0x10000>;
-		};
-
 		tpm5: tpm@40260000 {
 			compatible = "fsl,imx7ulp-tpm";
 			reg = <0x40260000 0x1000>;
-- 
1.7.9.5

